src/components/route-list/RoutesTable.tsx

```diff
@@ -93,32 +94,37 @@ const RoutesTable: React.FC<RoutesTableProps> = ({
                                     })}
                                 </ul>
                             </DataTableCell>
-                            <DataTableCell
-                                align="right"
-                                style={{ display: 'flex', gap: 10 }}
-                            >
-                                <Switch
-                                    onChange={({ checked: enabled }) =>
-                                        toggleRoute(route, !enabled)
-                                    }
-                                    checked={!route.disabled}
-                                />
-                                <Button
-                                    icon={<IconLaunch16 />}
-                                    small
-                                    onClick={() => showTestRouteModal(route)}
-                                >
-                                    {i18n.t('Test route')}
-                                </Button>{' '}
-                                <RouteActions
-                                    showSharingDialog={() =>
-                                        showSharingDialog(route)
-                                    }
-                                    showEditRouteModal={() =>
-                                        showEditRouteModal(route)
-                                    }
-                                    deleteRoute={() => deleteRoute(route.id)}
-                                />
+                            <DataTableCell align="right">
+                                <div className={styles.routeActionsCell}>
+                                    <Switch
+                                        onChange={({ checked: enabled }) =>
+                                            toggleRoute(route, !enabled)
+                                        }
+                                        checked={!route.disabled}
+                                        label={i18n.t('Enable route')}
```

Comment:
Should this be an inline label in the row or would it be better as a column header?

Reply:
@cooper-joe I will leave the Switch label as it is now, but let me know if you think moving it to a column header is better as Austin mentions (it does seem a bit crowded now)
 
![image](https://github.com/user-attachments/assets/5309a4f5-034d-4a84-b856-24cd17d550f1)